crates/eth-rpc/src/api/kakarot_custom_api.rs

```diff
@@ -0,0 +1,14 @@
+use jsonrpsee::proc_macros::rpc;
+use kakarot_rpc_core::models::balance::TokenBalances;
+use reth_primitives::Address;
+
+#[rpc(server)]
+#[async_trait]
+pub trait KakarotCustomApi {
+    #[method(name = "kakarot_getTokenBalances")]
+    async fn token_balances(
+        &self,
+        address: Address,
+        contract_addresses: Vec<Address>,
+    ) -> jsonrpsee::core::RpcResult<TokenBalances>;
```

Comment:
can we keep the same semantic as in the other trait, ie import 

```
use jsonrpsee::core::RpcResult as Result;
```

and return

```
Result<TokenBalances>
```

Reply:
Done.